scripts/travis-run

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e  # Exit on non-zero status
+set -x  # Print commands run
+set -u  # Treat unset variables as an error
+
+# XXX would be nice to have code comments explaining each of these
+docker-compose --version
+docker-compose pull
+docker-compose build
+docker-compose up -d
+# XXX Sleep here?!
```

Comment:
You can use the Python package `urlwait` (https://pypi.org/project/urlwait/), which is baked into the `mdnwebdocs/kuma_base` image. In our Jenkins code (see the `compose_test()` function within `Jenkinsfiles/utils.groovy`), we use `docker-compose.test.yml` which waits on `mysql` to spin-up before running tests. So perhaps we could add the following to the Travis environment (by default `urlwait` waits to connect to the service defined in `DATABASE_URL`) :
```
- DATABASE_URL=mysql://root:kuma@mysql:3306/developer_mozilla_org
- URLWAIT_TIMEOUT=300
```
and then use 
```
docker-compose exec -T urlwait && npm run webpack
```